### PR TITLE
Add skia to component set, and remove dependency on --iso-8601

### PR DESCRIPTION
**Pull request by shannonwoods**
These are changes I made to be able to run locally on my Macbook with Skia included in the count. Not entirely certain I got the formatting correct on --iso-8601; I can't check it on my machine.

### CODE REVIEW

File: chrome_gpu_bug_count.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ BASEDIR=$(dirname "$0")
 INPUT="$BASEDIR/chrome_gpu_triage.json"
 SCRIPT="$BASEDIR/issue_counter.py"
 mkdir -p "$OUTDIR"
-OUTFILE="$OUTDIR/$(date --iso-8601).json"
+OUTFILE="$OUTDIR/$(date "+%Y-%m-%d-%H:%M").json"
```
Comment (quisquous) [inline chrome_gpu_bug_count.sh]: Ah! --iso-8601 for me is date "+%Y-%m-%d", just the date with no time.

**Review by quisquous — APPROVED** [review]

File: chrome_gpu_bug_count.sh (hunk 1 above)
Comment (shannonwoods) [reply chrome_gpu_bug_count.sh]: I'm happy to submit a CL to change it to that, if it'd make your logs/CSV upload process smoother!

Comment (quisquous) [reply chrome_gpu_bug_count.sh]: Yeah, if you don't mind updating your branch to have that, I'd appreciate it.  I find time to be kind of noisy in filenames.  If you'd prefer keeping time and are also using this script, I'm happy to change too.

Comment (shannonwoods) [reply chrome_gpu_bug_count.sh]: I think I updated the PR with the additional commit correctly? I don't GitHub much, so if I need to rewrangle, let me know.

**Comment by quisquous** [comment]
Thanks!

**Pull request merged**